themes/src/main/resources/theme/base/login/template.ftl

```diff
@@ -151,4 +151,19 @@
   </div>
 </body>
 </html>
+
+<script>
+    const forms = document.getElementsByTagName('form');
+
+    for (var i = 0; i < forms.length; i++) {
+        const form = forms[i];
+        form.addEventListener("submit", preventDoubleClick);
```

Comment:
Nit: we should be consistent in how we use strings.
```suggestion
        form.addEventListener('submit', preventDoubleClick);
```

Reply:
Done